.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
         "~/.local/lib/python3.8/site-packages"
     ],
     "python.analysis.autoImportCompletions": false,
-    "python.formatting.provider": "black"
+    "python.formatting.provider": "black",
+    "cmake.sourceDirectory": "/home/gdemoura/amiga-dev-kit/farm-ng-amiga"
```

Comment:
Don't push stuff like this that is specific to your machine/user!

Reply:
Sorry I didn't see this getting added to the commit. I probably missed it bc I was rushing to publish this doc for Jeff's call last Friday. I will remove it in a future PR.

Reply:
No worries, doing it in #214 

If you're constantly having to ignore that change (which you shouldn't be) we can adjust settings either for your vs code or using .gitignore